example_simulation/example.conf

```diff
@@ -32,27 +32,27 @@ kim-model = EAM_Dynamo_Ackland_1987_Au__MO_754413982908_00
 #         the specific material in the periodic table from: https://openkim.org/
 initial-temperature = 40 
 # Arg:    Mandatory
-# Type:   Float
-# Discrp: The intial temprature of the simulation.
+# Type:   Float or list of floats
+# Discrp: The intial temperature of the simulation.
 target-temperature = 40 
 # Arg:    Mandatory if ensemble is NVT, otherwise optional
-# Type:   Float
-# Discrp: The intial temprature of the simulation.
+# Type:   Float or list of floats
+# Discrp: The target temperature of the simulation.
 repeat = 0 
 # Arg:    Mandatory
-# Type:   Int
-# Discrp: The cellsize (generated by repeated sized of a unitcell) of the simulation.
+# Type:   Interger or list of integers
+# Discrp: The cellsize (generated by repeated sized of a single unitcell) of the simulation.
 #         Example, if repeat=0 then simulating a unitcell.
 steps = 100 
 # Arg:    Mandatory
-# Type:   Int
+# Type:   Interger or list of integers
 # Discrp: Number of timesteps for the simulation, 1 timestep=1fs
 use-asap = False
 # Arg:    Mandatory
-# Type:   Boolean
+# Type:   Boolean or list of booleans
 # Discrp: True if the user want to use Asap, otherwise False
 volume-scale = (a,b,c) 
 # Arg:    Optional
-# Type:   Tupple, a,b: float and c: int
+# Type:   Syntax, a,b: float and c: int
 # Discrp: A uniformed interval, [a-b, a+b], is generated with 
 #         c: number of uniformly spaced samples. 
```

Comment:
Write something about what volume-scale does (scales the volume by that factor). volume-scale = 1 doesn't change the volume. < 1 shrinks it, > 1 increases it.

Reply:
done